src/deserialize/conversion.rs

```diff
@@ -0,0 +1,61 @@
+use pyo3::sync::GILOnceCell;
+use pyo3::types::{PyAnyMethods, PyDict, PyInt, PyType};
+use pyo3::{Bound, IntoPyObject, Py, PyAny, PyErr, PyResult, Python, ffi};
+use scylla_cql::value::{CqlDuration, CqlVarintBorrowed};
+static RELATIVEDELTA_CLS: GILOnceCell<Py<PyType>> = GILOnceCell::new();
+
+fn get_relative_delta_cls(py: Python<'_>) -> PyResult<&Bound<'_, PyType>> {
+    RELATIVEDELTA_CLS.import(py, "dateutil.relativedelta", "relativedelta")
+}
+
+pub(crate) struct CqlVarintWrapper<'b> {
+    val: CqlVarintBorrowed<'b>,
+}
+
+impl<'b> From<CqlVarintBorrowed<'b>> for CqlVarintWrapper<'b> {
+    fn from(val: CqlVarintBorrowed<'b>) -> Self {
+        Self { val }
+    }
+}
+
+impl<'py> IntoPyObject<'py> for CqlVarintWrapper<'_> {
+    type Target = PyInt;
+    type Output = Bound<'py, Self::Target>;
+    type Error = PyErr;
+
+    fn into_pyobject(self, py: Python<'py>) -> Result<Self::Output, Self::Error> {
+        let bytes = self.val.as_signed_bytes_be_slice();
+        unsafe {
+            let val = ffi::_PyLong_FromByteArray(bytes.as_ptr(), bytes.len(), 0, 1);
+
+            Ok(Bound::from_owned_ptr(py, val).downcast_into()?)
+        }
+    }
+}
```

Comment:
:question: Where and how did you find this `ffi::_PyLong_FromByteArray`? I had trouble when perusing PyO3 docs.